src/NuGet.Clients/NuGet.VisualStudio.Client/source.extension.vsixmanifest

```diff
@@ -22,19 +22,23 @@
     </Installation>
     <Assets>
         <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:ProjectName="NuGet.Tools" Path="|NuGet.Tools;PkgdefProjectOutputGroup|" />
+        <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:ProjectName="NuGet.SolutionRestoreManager" Path="|NuGet.SolutionRestoreManager;PkgdefProjectOutputGroup|" />
+        <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" Path="|NuGet.VisualStudio.OnlineEnvironment.Client;PkgdefProjectOutputGroup|" />
+
+        <!-- in order to property be imported/exported, all assemblies which do that, need to be listed as a MefComponent here-->
```

Comment:
This comment seems unnecessary. Would the vsix designer preserve it?

Reply:
is there a vsix designer? how do i launch it?

Reply:
It's the default action for me in VS. 

![image](https://user-images.githubusercontent.com/2878341/80267277-b3caab00-8654-11ea-8c55-164af9baf39f.png)

Assets and Dependencies is where you can manage things.